lisp_ease/package.xml~

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0"?>
+<package>
+  <name>lisp_ease</name>
+  <version>0.0.0</version>
+  <description>The lisp_ease package</description>
+
+  <!-- One maintainer tag required, multiple allowed, one person per tag -->
+  <!-- Example:  -->
+  <!-- <maintainer email="jane.doe@example.com">Jane Doe</maintainer> -->
+  <maintainer email="hawkin@uni-bremen.de">Alina Hawkin</maintainer>
+
+
+  <!-- One license tag required, multiple allowed, one license per tag -->
+  <!-- Commonly used license strings: -->
+  <!--   BSD, MIT, Boost Software License, GPLv2, GPLv3, LGPLv2.1, LGPLv3 -->
+  <license>TODO</license>
+
+
+  <!-- Url tags are optional, but multiple are allowed, one per tag -->
+  <!-- Optional attribute type can be: website, bugtracker, or repository -->
+  <!-- Example: -->
+  <!-- <url type="website">http://wiki.ros.org/lisp_ease</url> -->
+
+
+  <!-- Author tags are optional, multiple are allowed, one per tag -->
+  <!-- Authors do not have to be maintainers, but could be -->
+  <!-- Example: -->
+  <!-- <author email="jane.doe@example.com">Jane Doe</author> -->
+
+
+  <!-- The *_depend tags are used to specify dependencies -->
+  <!-- Dependencies can be catkin packages or system dependencies -->
+  <!-- Examples: -->
+  <!-- Use build_depend for packages you need at compile time: -->
+  <!--   <build_depend>message_generation</build_depend> -->
+  <!-- Use buildtool_depend for build tool packages: -->
+  <!--   <buildtool_depend>catkin</buildtool_depend> -->
+  <!-- Use run_depend for packages you need at runtime: -->
+  <!--   <run_depend>message_runtime</run_depend> -->
+  <!-- Use test_depend for packages you need only for testing: -->
+  <!--   <test_depend>gtest</test_depend> -->
+  <buildtool_depend>catkin</buildtool_depend>
+  <build_depend>cram_json_prolog</build_depend>
+  <build_depend>json_prolog</build_depend>
+  <build_depend>roslisp</build_depend>
+  <run_depend>cram_json_prolog</run_depend>
+  <run_depend>json_prolog</run_depend>
+  <run_depend>roslisp</run_depend>
+
+
+  <!-- The export tag contains other, unspecified, tags -->
+  <export>
+    <!-- Other tools can request additional information be placed here -->
+
+  </export>
+</package>
```

Comment:
Please remove Koelln...jpg and axes.blend and -1 files.

Reply:
ba_axes -> axes
bowl -> edeka_red_bowl
cup -> cup_eco_orange
fork -> fork_blue_plastic
spoon -> spoon_blue_plastic
muesli -> koelln_muesli_honig_nuss
milk -> weide_milch_small or so.

Reply:
3D model convention:

(0) origin is center of geometry, bounding box
(1) z points upwards in a natural state of object standing (lying) on a surface
(2) x is along the longer dimension of object
(3) x points from handle or / and to head
(4) y points towards logo if one present
(5) remaining x or y axis is decided by right hand rule